kotlin-testing/src/main/kotlin/com/baeldung/argumentcaptor/OrderService.kt

```diff
@@ -0,0 +1,33 @@
+package com.baeldung.argumentcaptor
+
+class OrderService(
+    private val orderRepository: OrderRepository,
+    private val messageDispatcher: IMessageDispatcher,
+) {
+
+    fun createOrder(order: Order, confirmRequired: Boolean) {
+
+        val orderId = orderRepository.createNewOrder(order)
+
+        if (confirmRequired) {
+            messageDispatcher.sendMessage("Order Created") { success ->
+                if (success) {
+                    orderRepository.updateOrderConfirmation(orderId, true);
+                }
+                else {
+                    throw IllegalStateException("Failed to send confirmation message!")
+                }}
+        }
+        else {
```

Comment:
Is this spacing of `else` common for Kotlin?

Reply:
Changed it to be consistent with Kotlin standard